### DISCUSSION TITLE
Need help with multiple stores

### DISCUSSION THREAD

**Discussion by imranalii** [osiset/laravel-shopify#711] [Q&A]
@osiset thanks for providing one of best package!
I am new to this package & need help regarding following:
- I want to link multiple stores for each user/store owner.
- For example if user has 2 stores, I want to get data for all 2 stores in current session.
- For now I can get store data for current logged-in store only.
- How can I get other store data in current session.
Please help in this regard, thanks.

**Comment by gnikyt** [comment]
This is nothing the package can do for you, youd have to do this manually.

If you know the other store, simply do a call to get the data for that store...

```php
$currentStore = Auth::user();
$otherStore = User::where('name', 'other-store.myshopify.com')->first();

$currentStoreInfo = $currentStore->api()->rest('GET', '/admin/shop.json');
$otherStoreInfo = $otherStore->api()->rest('GET', '/admin/shop.json');
```

**Reply by imranalii** [reply]
Thanks a lot for your quick reply.
